shared/local-debug.desktop.js

```diff
@@ -41,24 +35,37 @@ if (__DEV__ && process.env.KEYBASE_LOCAL_DEBUG) { // eslint-disable-line no-unde
   config.enableActionLogging = false
   config.forwardLogs = true
   config.devStoreChangingFunctions = true
-  config.resizeLoginForm = false
+  config.printOutstandingRPCs = true
 }
 
 config = updateConfig(config)
 
 export const {
   enableActionLogging,
-  overrideRouterState,
-  overrideActiveTab,
   skipLoginRouteToRoot,
   allowStartupFailure,
   printRPC,
   showDevTools,
-  showMainWindow,
   showAllTrackers,
   reduxDevToolsSelect,
   enableStoreLogging,
   forwardLogs,
   devStoreChangingFunctions,
-  resizeLoginForm
+  printOutstandingRPCs
 } = config
+
+export function initTabbedRouterState (state) {
```

Comment:
instead of setting a starting tab and the uri (for all tabs, a known issue) we know just override various pieces in this function. we could discuss just a general store mutator or something...